### PR TITLE
78 avalanche subnet docs

### LINKED ISSUE #78: Add `avalancheSubnet` resource documentation

### PR DESCRIPTION
**Pull request by leopaul36**
### Linked issues

- FIx #78

### Changes

- Add "Subnet" page to "Avalanche Devnet" guide
- Add Subnet blueprint sections to "One-Command Devnet" guide

### CODE REVIEW

File: docs/console/guides/blueprint/blueprint-configuration.md [hunk 1]
```diff
@@ -221,6 +259,106 @@ Resource successfully created in project 'ash-devnet'!
...
+    ID:      2SZGABMprnB9Ux88WLYGodeWRFHJBogDrkncLDqsv5XhFnizT6
+    VM ID:   srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+    VM type: SubnetEVM
+  Validators list (5): 
+  - NodeID-MFrZFVCXPv5iCn6M9K6XduxGTYp891xXZ
+  - NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5
+  - NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg
+  - NodeID-GWPcbFJZFfZreETSoWjPimr846mXEKCtu
+  - NodeID-NFBbbJ4qCmNaCzeW7sxErhvWqvEQMnYcN
+```
```
Comment (Nuttymoon) [inline docs/console/guides/blueprint/blueprint-configuration.md:360]: ## Blockchain RPC endpoint

The CLI provides a powerful helper to get the RPC endpoint that can be used to query the Subnet EVM blockchain (e.g. to connect a Web3 wallet). Provide the `avalancheNode` resource to be used as RPC and the `avalancheSubnet`:

```bash title="Command"
ash console helper rpc local-node-01 my-subnet-evm
```

```bash title="Output"
AshLocalEVM RCP endpoint:
  http://44.223.28.33:9650/ext/bc/RfX8YRUHePxFENBZFwaNt1tP6CR6RQ4hJJ5AqYv49LRoSKniK/rpc
```

Comment (Nuttymoon) [reply docs/console/guides/blueprint/blueprint-configuration.md:360]: Cannot do a suggestion on a README with code bocks

File: docs/console/guides/blueprint/blueprint-configuration.md [hunk 2]
```diff
@@ -221,6 +259,106 @@ Resource successfully created in project 'ash-devnet'!
...
+
+```bash title="Output"
+Resource 'ash-subnet' of project 'ash-devnet':
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------------------------------------------------------+
+| Resource name | Resource ID | Type            | Cloud region  | Size  | Created at       | Status  | Resource specific                                                 |
++===============+=============+=================+===============+=======+==================+=========+===================================================================+
+| ash-subnet    | 396d...6cc8 | AvalancheSubnet | aws/us-east-1 | Small | 2024-01-30T17:26 | Running |  ID         | 2rArnpKkN9fKch9Q6ana3awAfVmM4HfLoNYrZZDkSumApEi8yZ  |
+|               |             |                 |               |       |                  |         |  Validators | 5                                                   |
++---------------+-------------+-----------------+---------------+-------+------------------+---------+-------------------------------------------------------------------+
+```
```
Comment (Nuttymoon) [inline docs/console/guides/blueprint/blueprint-configuration.md:334]: ## Subnet information

After a few minutes, we can get the Subnet ID, control keys, blockchains, and validator nodes information from the `subnetStatus` field returned by the `console resource info` command:

```bash title="Command"
ash console resource info my-subnet-evm --json | jq '.subnetStatus'
```

:::tip
We use the `--json` flag to get the extended information about the Subnet. All those information cannot be displayed in a table format.
:::

```python title="Output"
{
  "blockchains": [
    {
      "id": "RfX8YRUHePxFENBZFwaNt1tP6CR6RQ4hJJ5AqYv49LRoSKniK",
      "name": "AshLocalEVM",
      "vm_id": "srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy",
      "vm_type": "SubnetEVM"
    }
  ],
  "controlKeys": [
    "P-local18jma8ppw3nhx5r4ap8clazz0dps7rv5u00z96u"
  ],
  "id": "29uVeLPJB1eQJkzRemU8g8wZDw5uJRqpab5U2mX9euieVwiEbL",
  "pendingValidators": [],
  "subnetType": "Permissioned",
  "threshold": 1,
  "validators": [
    {
      "connected": true,
      "end_time": 1707486609,
      "node_id": "NodeID-MFrZFVCXPv5iCn6M9K6XduxGTYp891xXZ",
      "stake_amount": 100,
      "start_time": 1706881809,
      "tx_id": "DhEgYijBur6QDUDgUMpmDAyF5XGyq6JwG919B8bkryDLijvua",
      "uptime": 100,
      "weight": 100
    },
    # ...
  ]
}
```

Comment (Nuttymoon) [reply docs/console/guides/blueprint/blueprint-configuration.md:334]: Cannot do a suggestion on a README with code bocks

File: docs/console/guides/local-network/subnet.md [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+    ID:      2VgLztfZWQewKVqtsBVpdbMdEFWScDgcwP21UWQwifomCG8weF
+    VM ID:   srEXiWaHuhNyGwPUi444Tu47ZEDwxTWrbQiuD7FmgSAQ6X7Dy
+    VM type: SubnetEVM
+  Validators list (5): 
+  - NodeID-7Xhw2mDxuDS44j42TCB6U5579esbSt3Lg
+  - NodeID-GWPcbFJZFfZreETSoWjPimr846mXEKCtu
+  - NodeID-NFBbbJ4qCmNaCzeW7sxErhvWqvEQMnYcN
+  - NodeID-MFrZFVCXPv5iCn6M9K6XduxGTYp891xXZ
+  - NodeID-P7oB2McjBGgW2NXXWVYjV8JEDFoW9xDE5
+```
```
Comment (Nuttymoon) [inline docs/console/guides/local-network/subnet.md:173]: Same comments as above

**Review by Nuttymoon — CHANGES_REQUESTED** [review]
We should also talk about nodes' restart to apply the changes. Even though I think you are right, nodes restart automatically when they shouldn't.

**Comment by Nuttymoon** [comment]
I think there is also a naming drift in the `avalancheSubnet` resource name between the 2 guides

File: docs/console/guides/local-network/subnet.md [hunk 4]
```diff
@@ -0,0 +1,173 @@
...
+    local-node-04,
+    local-node-05,
+  ],
+  subnetConfig: {
+    avalancheSubnetConfig: {
+      "subnet_blockchains_list": [
+          {
+            "name": "AshLocalEVM",
+            "vm": "subnet-evm",
+            "genesis_data": {
```
[the remark this answers is not in the document]
Comment (leopaul36) [reply docs/console/guides/local-network/subnet.md]: Done in https://github.com/AshAvalanche/ash-docs/pull/79/commits/6df71bc71fe5f55f263dfd532ef4b6b885d72d2b

File: docs/console/guides/blueprint/blueprint-configuration.md (hunk 1 above)
Comment (leopaul36) [reply docs/console/guides/blueprint/blueprint-configuration.md]: Done in https://github.com/AshAvalanche/ash-docs/pull/79/commits/6df71bc71fe5f55f263dfd532ef4b6b885d72d2b

File: docs/console/guides/blueprint/blueprint-configuration.md (hunk 2 above)
Comment (leopaul36) [reply docs/console/guides/blueprint/blueprint-configuration.md]: Done in https://github.com/AshAvalanche/ash-docs/pull/79/commits/6df71bc71fe5f55f263dfd532ef4b6b885d72d2b

File: docs/console/guides/local-network/subnet.md (hunk 3 above)
Comment (leopaul36) [reply docs/console/guides/local-network/subnet.md]: Done in https://github.com/AshAvalanche/ash-docs/pull/79/commits/6df71bc71fe5f55f263dfd532ef4b6b885d72d2b

**Comment by leopaul36** [comment]
> I think there is also a naming drift in the `avalancheSubnet` resource name between the 2 guides

Done in

**Review by Nuttymoon — APPROVED** [review]

**Pull request merged**